core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -256,15 +287,10 @@ private[spark] class BlockManager(
 
   var hostLocalDirManager: Option[HostLocalDirManager] = None
 
-  private lazy val migratableResolver: MigratableResolver = {
+  private[storage] lazy val migratableResolver: MigratableResolver = {
```

Comment:
I'm wondering why you chose to leave this here since I think it's only accessed by the `BlockManagerDecommissionManager`.

Reply:
You are right, we do not need this member after the refactoring.

Reply:
Ah actually we do need it you were right originally (it's also used in doPut)